spec/requests/users_controller_spec.rb

```diff
@@ -1,6 +1,10 @@
 require "rails_helper"
 
 RSpec.describe UsersController, type: :request do
+  before do
+    ActiveResource::HttpMock.reset!
```

Comment:
I don't understand why this is necessary

Reply:
This was because I had these tests passing in isolation but failing when run alongside all other tests. The failure was related to request mocks existing in this context despite being set up in other tests. I'm hoping this is a red herring and something peculiar to my working copy at the time, as I've not been able to reproduce the same situation since. I've removed this reset now.